apps/web/src/app/admin/students/enrollments/layout.tsx

```
@@ -0,0 +1,30 @@
+import { redirect } from "next/navigation";
+
+import { getUser } from "@/queries/getUser";
+
+export default async function EnrollmentsLayout({
+	children,
+}: {
+	children: React.ReactNode;
+}) {
+	let session;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)